docs/public/specifications/test-zones/Address-TP/address01.md

```diff
@@ -0,0 +1,206 @@
+# Specification of test scenarios for ADDRESS01
+
+
+## Table of contents
+
+* [Background](#background)
+* [Test Case](#test-case)
+* [Test scenarios](#test-scenarios)
+* [Test zone names](#test-zone-names)
+* [Test scenarios and message tags](#test-scenarios-and-message-tags)
+* [Zone setup for test scenarios]
+
+
+## Background
+
+See the [test zone README file].
+
+## Test Case
+This document specifies defined test scenarios for test case [ADDRESS01].
+
+## Test scenarios
+
+The purpose of the test scenarios is to cover all reasonable contexts where
+different message tags are outputted when [ADDRESS01] is run on a test zone.
+The message tags are defined in the test case ([ADDRESS01]) and the scenarios
+are defined below.
+
+The test scenarios are structured as stated in the [test zone README file].
+
+## Test zone names
+
+The test zone for each test scenario in this document is a subdomain delegated
+from the base name (`address01.xa`) and that subdomain having the same name as the
+scenario. The names of those zones are given in section
+"[Zone setup for test scenarios]" below.
+
+## All tags
+
+The test case can output any of these message tags, but not necessarily in any combination.
+
+- A01_ADDR_GLOBALLY_REACHABLE    
+- A01_ADDR_NOT_GLOBALLY_REACHABLE
+- A01_DOCUMENTATION_ADDR
```

Comment:
The tag `A01_ADDR_GLOBALLY_REACHABLE` was changed to `A01_GLOBALLY_REACHABLE_ADDR` in the test specification. (+ reordering)
```suggestion
- A01_ADDR_NOT_GLOBALLY_REACHABLE
- A01_DOCUMENTATION_ADDR
- A01_GLOBALLY_REACHABLE_ADDR
```